stable/falco/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 This file documents all notable changes to Sysdig Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.0
```

Comment:
Use 1.0.1 instead of 1.1.0

Thanks!

Reply:
Yeah sure, have pushed the change, thanks!